ydb/core/blobstorage/pdisk/blobstorage_pdisk_color_limits.h

```diff
@@ -49,16 +49,26 @@ struct TColorLimits {
         str << "  Cyan = Total * " << Cyan.ToString() << "\n";
     }
 
-    static TColorLimits MakeChunkLimits() {
+    static TColorLimits MakeChunkLimits(i64 cyan = 130) {
```

Comment:
```suggestion
    static TColorLimits MakeChunkLimits(i64 cyan) {
```

мне кажется дефолт тут лишний, так как используется функция в одном месте и там всегда указывается аргумент